packages/app/src/layouts/default.vue

```diff
@@ -98,6 +122,13 @@ const showHeader = computed(() => {
   return currentRoute.meta.header !== false
 })
 
+const showEnableNotificationsBanner = computed(() => {
+  return userProjectStatusStore.cloudStatus === 'allTasksCompleted' &&
+    query.data.value?.localSettings.preferences.desktopNotificationsEnabled === null && (
+    query.data.value?.localSettings.preferences.dismissNotificationBannerUntil ?
+      Date.now() > new Date(query.data.value?.localSettings.preferences.dismissNotificationBannerUntil).getTime() : true)
```

Comment:
yeah, I think I originally did it just because I was using `dayjs` to correctly add 3 days to the current time, and I just threw that object in the `state.json` file.

I'll update this to save the setting in milliseconds instead

Reply:
Eh, I tried this and ran into some issues... I think it's fine to use a date here. We can revisit this before we merge the feature branch if you feel strongly about it